apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -105,14 +112,29 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
       <VersionHistoryButton startSources={startSources} appName={appName} />
       {appName === 'pythonlab' && (
         <WithTooltip tooltipProps={feedbackTooltipProps}>
-          <Button
+          <LinkButton
             isIconOnly
             icon={{iconStyle: 'solid', iconName: 'commenting'}}
-            onClick={openFeedbackForm}
+            href={'https://forms.gle/Z4FsGMFzE4NrFp369'}
             ariaLabel={commonI18n.feedback()}
             size={'xs'}
             type={'tertiary'}
             className={darkModeStyles.tertiaryButton}
+            target="_blank"
+          />
+        </WithTooltip>
+      )}
+      {/* For now, only python lab supports documentation */}
+      {appName === 'pythonlab' && (
```

Comment:
super nit: Checking for `pythonlab` in two places now. Could use helper function like `isPythonlab`.